libapicore/ApiServer.cpp

```diff
@@ -142,63 +143,101 @@ void ApiConnection::processRequest(Json::Value& requestObject)
 	std::string _method = requestObject.get("method", "").asString();
 	jRes["id"] = requestObject.get("id", 0).asInt();
 
+	// Check authentication
+	if (!m_is_authenticated) {
```

Comment:
Have you tried integrate clang-format to your editor. It would make it easier to have consistent coding style.

Reply:
Ok for formatting.